packages/browser-wallet/src/shared/storage/types.ts

```diff
@@ -158,3 +159,22 @@ export interface NetworkConfiguration {
     jsonRpcUrl: string;
     explorerUrl: string;
 }
+
+export type RecoveryPayload = {
+    providers: IdentityProvider[];
+    globalContext: CryptographicParameters;
+    seedAsHex: string;
```

Comment:
You already have the session password, so you should decrypt the seed phrase instead of saving it again here.